Take02/wwwroot/js/site.js

```diff
@@ -120,14 +123,22 @@ function wireNextLink(anchorTag) {
         $(ingredient).remove();
         $(ingredient).insertAfter(next);
 
+        swapOrder(ingredient, next);
+
         wireOrderingLinks();
     });
 }
 
+function swapOrder(first, second) {
```

Comment:
This grabs the `<input name="number" >` element from each ingredient and switches the `val` of them